features/docker/pipework

```diff
@@ -0,0 +1,248 @@
+#!/bin/sh
```

Comment:
did you write this? no license, no author (and no rpm?)

Reply:
Looks a lot like an earlier version of https://github.com/jpetazzo/pipework/blob/master/pipework to me.

Reply:
that's licensed under APL2.0, so should be fine to add the source and mention it in the header that is apl